x-pack/plugins/security_solution/server/lib/detection_engine/signals/build_bulk_body.ts

```diff
@@ -121,10 +121,9 @@ export const buildSignalFromSequence = (
 ): SignalHit => {
   const rule = buildRuleWithoutOverrides(ruleSO);
   const timestamp = new Date().toISOString();
-
-  const reason = buildReasonMessage({ rule });
-  const signal: Signal = buildSignal(events, rule, reason);
   const mergedEvents = objectArrayIntersection(events.map((event) => event._source));
+  const reason = buildReasonMessage({ rule, mergedDoc: mergedEvents as SignalSourceHit });
```

Comment:
i struggled with this typecast for a while, couldn't get it either. blerg

Reply:
Yea, it was a pain. Just went with the path of least resistance 😅